packages/kbn-optimizer/limits.yml

```diff
@@ -1,163 +1,164 @@
 pageLoadAssetSize:
-  actions: 20000
-  advancedSettings: 27596
-  aiops: 10000
-  alerting: 106936
+  actions: 88650
```

Comment:
Can we limit changes here to the uptime plugin?

Reply:
Done @jbudz :)